src/wp-includes/class-wp-scripts.php

```diff
@@ -294,15 +294,42 @@ public function do_item( $handle, $group = false ) {
 			$cond_after  = "<![endif]-->\n";
 		}
 
+		$strategy = $this->get_eligible_loading_strategy( $handle );
+
 		$before_handle = $this->print_inline_script( $handle, 'before', false );
-		$after_handle  = $this->print_inline_script( $handle, 'after', false );
 
 		if ( $before_handle ) {
 			$before_handle = sprintf( "<script%s id='%s-js-before'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $before_handle );
 		}
 
-		if ( $after_handle ) {
-			$after_handle = sprintf( "<script%s id='%s-js-after'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $after_handle );
+		$after_handle = '';
+		if ( '' === $strategy ) {
+			$after_handle = $this->print_inline_script( $handle, 'after', false );
+
+			if ( $after_handle ) {
+				$after_handle = sprintf( "<script%s id='%s-js-after'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), $after_handle );
```

Comment:
@joemcgill This one won't work and I am not able to get a solution for this. Tried a couple of solution suggested online. 

Issues is `$s` in numbered placeholder is considered as a variable when when we used double quotes for parent string and `\n` is taken as a string instead of new line literal when I use single quotes for parent string. So for that reason I have kept it normal(one without number) for smaller string with non repeating attributes. And for larger string, which has some attribute repeating again and again, i used placeholder but instead of `\n` I had to use `PHP_EOF` as `\n` was not working for me even when passed as attribute.

Reply:
@joemcgill  
`'<script%1$s id="%2$s-js-after">\n%3$%s\n</script>\n'` this wont work 
as `\n` doesn't work with a single quote enclosure. In this case placeholder will work by not `\n`

I can do the following to support `\n`, but then `$s` will be considered as a variable. 
`"<script%1$s id="%2$s-js-after">\n%3$%s\n</script>\n"`

Reply:
@kt-12 this can be solved by escaping the `$` as follows:

```
"<script%1\$s id="%2\$s-js-after">\n%3\$s%s\n</script>\n"
```

Reply:
@10upsimon It worked Thanks. Maybe i did make some mistake initially.  

@joemcgill  I have updated at all places.